src/Sarif.Driver.UnitTests/TestAnalysisContext.cs

```diff
@@ -1,32 +1,10 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-
 namespace Microsoft.CodeAnalysis.Sarif.Driver
 {
-    public class TestAnalysisContext : IAnalysisContext
+    public class TestAnalysisContext : AnalysisContext
```

Comment:
There's no need to derive directly from `IAnalysisContext` and reimplement all the members. `AnalysisContext` already does that.

Reply:
nice

---
In reply to: [161965854](https://github.com/Microsoft/sarif-sdk/pull/752#discussion_r161965854) [](ancestors = 161965854)